src/components/Photography/SeeMoreLink.tsx

```diff
@@ -0,0 +1,53 @@
+import Link from 'gatsby-link'
+import React from 'react'
+import { Box, Flex, Text } from 'rebass/styled-components'
+
+const SeeMoreLink = ({
+  href,
+  totalNumImages,
+}: {
+  href: string
+  totalNumImages: number
+}) => {
+  if (totalNumImages <= 6) {
+    return (
+      <Box
+        marginBottom={2}
+        marginTop={4}
+        style={{ borderTop: '1px solid rgb(221, 221, 221)' }}
+      >
+        &nbsp;
+      </Box>
+    )
+  } else {
+    return (
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4. (@typescript-eslint/indent)](https://app.codacy.com/app/jdstuart/jesses.io/pullRequest?prid=4576429)